docs/application-tutorial/game_of_life.md

```diff
@@ -0,0 +1,54 @@
+# Game of Life
+
+In the associated [Python file](https://github.com/zama-ai/concrete/blob/main/frontends/concrete-python/examples/game_of_life/game_of_life.py), you can run the Game of Life, written in Concrete Python.
+
+![ Game of Life](../../docs/_static/game_of_life.gif)
```

Comment:
what's the right way to have the path of the file, @umut-sahin , please? ideally which works with GitBook and GitHub. CC @aquint-zama 

(was easier when everything was at the same place, in my opinion)

Reply:
```
![ Game of Life](../_static/game_of_life.gif)
```

Reply:
thx done